src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -58,15 +58,7 @@ IOptions<DependencyUpdateErrorProcessorOptions> options
         [CronSchedule("0 0 0/1 1/1 * ? *", TimeZones.PST)]
         public async Task ProcessDependencyUpdateErrorsAsync()
         {
-            if (_options.ConfigurationRefresherEndPointUri == null && _options.DynamicConfigs == null)
-            {
-                _logger.LogInformation("Dependency Update Error processor is disabled because no App Configuration was available.");
-                return;
-            }
-            await _options.ConfigurationRefresherEndPointUri.Refresh();
-            if (bool.TryParse(_options.DynamicConfigs["FeatureManagement:DependencyUpdateErrorProcessor"],
-                out var dependencyUpdateErrorProcessorFlag) &&
-                dependencyUpdateErrorProcessorFlag)
+            if(_options.IsEnabled)
```

Comment:
Removed this code here, cos Alex fixed the feature flag calling part. So I can directly call featureflag from config.

Reply:
I would make that a separate PR, Is it related to the test infrastructure?

Reply:
okay let me quickly create a new pr for that :)

Reply:
I think it's fine to keep it here, but it's probably a good idea to call out that you're adding these changes to the PR description.

I didn't know if you actually intended these to be part of the PR, or were left overs from local testing.

Reply:
I moved that to a different pr. Resolving this